src/api/app/views/webui/kiwi/images/_preferences.html.haml

```diff
@@ -7,11 +7,15 @@
         %strong Image type:
         %span.fill{ data: { tag: 'type_image' }}= @image.preference.type_image
       %li
-        %strong Container config name:
-        %span.fill{ data: { tag: 'type_containerconfig_name' }}= @image.preference.type_containerconfig_name
-      %li
-        %strong Container config tag:
-        %span.fill{ data: { tag: 'type_containerconfig_tag' }}= @image.preference.type_containerconfig_tag
+        %strong Version:
+        %span.fill{ data: { tag: 'version' }}= @image.preference.version
```

Comment:
What a about a hint text, this is from the kiwi documentation:

>  The image version must be a three-part version number of the format: Major.Minor.Release.

Reply:
@ChrisBr I added some text, please re-review!

![screenshot from 2017-11-27 09-30-06](https://user-images.githubusercontent.com/9274/33259739-c0ebf92e-d355-11e7-9b1d-3548c2565d83.png)